From e07a6173e239a8ee81c1e916e721e58b87285ac7 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
Date: Thu, 12 Apr 2018 00:18:51 +0700
Subject: [PATCH 479/504] usb: renesas_usbhs: Add compatibility for r8a7744

This driver supports other SoCs, but they need boards/Soc depend code.
So, this patch adds device tree support for r8a7744 initially.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Kha Tran <kha.tran.px@rvc.renesas.com>
Signed-off-by: Quynh Nguyen <quynhnguyen.xb@renesas.com>
---
 drivers/usb/renesas_usbhs/common.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/usb/renesas_usbhs/common.c b/drivers/usb/renesas_usbhs/common.c
index 9884376..06b687e 100644
--- a/drivers/usb/renesas_usbhs/common.c
+++ b/drivers/usb/renesas_usbhs/common.c
@@ -473,6 +473,10 @@ static const struct of_device_id usbhs_of_match[] = {
 		.data = (void *)USBHS_TYPE_RCAR_GEN2,
 	},
 	{
+		.compatible = "renesas,usbhs-r8a7744",
+		.data = (void *)USBHS_TYPE_RCAR_GEN2,
+	},
+	{
 		.compatible = "renesas,usbhs-r8a7794",
 		.data = (void *)USBHS_TYPE_RCAR_GEN2,
 	},
-- 
2.7.4

